docs/docsite/rst/conf.py

```diff
@@ -65,6 +66,7 @@
     'notfound.extension',
     'sphinx_antsibull_ext',  # provides CSS for the plugin/module docs generated by antsibull
     'sphinx_copybutton',
+    '_ext.eol_version',
```

Comment:
It's better to be explicit about which extensions are local: https://github.com/ansible/awx-plugins/blob/0d569b5/docs/conf.py#L81C5-L81C26
```suggestion

    # In-tree extensions:
    'eol_version',  # conditionally injects a banner if EOL
```